src/Protocol/Protocol_1_13.cpp

```diff
@@ -107,7 +107,49 @@ void cProtocol_1_13::SendBlockChanges(int a_ChunkX, int a_ChunkZ, const sSetBloc
 
 void cProtocol_1_13::SendMapData(const cMap & a_Map, int a_DataStartX, int a_DataStartY)
 {
-	// TODO
+	ASSERT(m_State == 3);  // In game mode?
+
+	cPacketizer Pkt(*this, pktMapData);
+	Pkt.WriteVarInt32(a_Map.GetID());
+	Pkt.WriteBEUInt8(static_cast<UInt8>(a_Map.GetScale()));
+
+	// TODO: don't hardcode it.
+	// this thing shows the map icons.
+	// for example: the player pointer, fortress, treasure marker..
+	// more info at wiki.vg
```

Comment:
Yep, this comment doesn't really give me much enlightenment. Either take it away or properly describe what's going on - i.e. what is "this thing" and give a proper link to the actual section in wiki.vg.